demo/index.html

```diff
@@ -32,13 +32,15 @@
     </style>
     <main></main>
 
-    <script type="module">
-      import 'https://unpkg.com/marked@latest/marked.min.js';
-      import 'https://unpkg.com/prismjs@latest/prism.js';
+    <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/prismjs/prism.js"></script>
+    <script>
+      // import 'https://unpkg.com/marked@latest/marked.min.js';
+      // import 'https://unpkg.com/prismjs@latest/prism.js';
```

Comment:
@jason-capsule42  and @fajar-apri-alaska not sure why this was passed though? Why were lines 38 and 39 commented out and 35 and 36 added? This is not how we have things laid out in the generator. 

https://github.com/AlaskaAirlines/WC-Generator/blob/4a488a19c82508e4882626de03474ac555e1ba6a/template/demo/index.html#L31-L36

Reply:
I think this one was my bad, I should just add `<script type=module>` instead of did above changes. I forgot that this changes is not mandatory, just the `marked.parse` part that is.